app/src/main/java/org/fossasia/susi/ai/rest/ClientBuilder.java

```diff
@@ -95,6 +95,8 @@ public static SusiService getSusiApi() {
     }
 
     public static Call<ListSkillMetricsResponse> fetchListSkillMetricsCall(SkillMetricsDataQuery queryObject) {
+        if (susiService == null)
+            createSusiService();
```

Comment:
This issue is waiting to pop up again, rather than having `createSusiService`, add a getter which safely returns initialized service and use it everywhere

Reply:
@iamareebjamal we already have getSusiApi() method, so maybe using that to return the calls should be fine

Reply:
also @iamareebjamal I feel is there a need to put this Call in ClientBuilder when we can simply use